app/components/atoms/link/link.component.tsx

```diff
@@ -0,0 +1,21 @@
+import {Link as RemixLink} from '@remix-run/react';
+import {FC, ReactNode} from 'react';
+
+interface Props {
+  to: string;
+  children: ReactNode;
+}
+
+export const Link: FC<Props> = ({to, children}) => {
+  const isExternalLink = to.startsWith('http');
```

Comment:
Is this really the way you know if its external? lol

Reply:
I asked Tony about how to implement this and he said that this approach would be enough to not overcomplicate it